wiki/Community/Forum/en.md

```diff
@@ -0,0 +1,71 @@
+---
+tags:
+  - forums
+  - subforum
+---
+
+# Forum
+
+*See also: [Internet forum](https://en.wikipedia.org/wiki/Internet_forum)*
+
+[The osu! **forums**](https://osu.ppy.sh/community/forums) are a place for conversations where messages are grouped by subjects, and no quick response is expected. The discussions are searchable, stored forever, and may be revisited at any time, which makes them different from the near-instant messages in [osu! chat](/wiki/Community/Internet_Relay_Chat). As a result, forum messages are longer, contain more information, and typically provide more details than chat messages would.
+
+The osu! forums provide [features](#features) similar to most internet forums: [BBCode](/wiki/BBCode) markup, polls, pins and subscriptions, private messages, and [moderation](#moderation).
+
+## Structure
+
+Each of the osu! forums contains a variety of topics, called **threads**, as well as smaller categories, or **subforums**. Most of the forums are English only, but there also exists a number of [language-specific](https://osu.ppy.sh/community/forums#forum-23) forums for local communities that are large enough.
+
+While some forums, such as [Mapping Discussion](https://osu.ppy.sh/community/forums/56), have specific requirements or guidelines, all of them follow [the community rules](/wiki/Rules).
+
+## Roles and user groups
+
+Some users may belong to one or several custom groups, based on their community tasks, access rights, or past activity. The list of user groups can be found at [The Team § User groups](/wiki/People/The_Team#user-groups).
+
+In addition to profile badges provided by group membership, users may have [custom forum titles](/wiki/User_title) displayed under the avatar in each of their posts.
+
+### Moderation
+
+*See also: [Reporting bad behaviour § Forum](/wiki/Reporting_Bad_Behaviour#forum)*
+
+[Moderators](/wiki/People/The_Team/Global_Moderation_Team) are users who keep the forums clean and appropriate, and answer concerns related to that. Moderators can edit or delete inappropriate posts or threads, pin threads and move them between different forums, and temporarily prevent offenders from posting by [silencing](/wiki/Silence) them.
+
+## Features
```

Comment:
i can't imagine this being useful to any reader and it seems odd to have such a selective list here anyway. whole section makes it seem like the forums are separate software than the rest of the website (same with "custom forum engine" later in this article)

imo this section can just be removed

Reply:
fair, it doesn't have anything special